eventtracking/django/migrations/0001_initial.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 2.2.13 on 2020-07-26 09:33
+
+from django.db import migrations, models
```

Comment:
@feanil we are following the [OEP-26](https://open-edx-proposals.readthedocs.io/en/latest/oep-0026-arch-realtime-events.html#filter-processor) for the development and its mentioned in the OEP to provide Admin UI for configurations.

If we don't want it to be this way, the OEP also needs to be modified.

Reply:
@danialmalik thanks for the pointer, I think we can update the OEP.  Since the self-service aspect is part of future work, I believe we can defer any models until we're ready to build out the APIs and UI to allow for self-service.  While the administration is being done by site administrators, I believe we should use django settings which are designed to be more easily managed by site admins.

Reply:
I've got a PR up on the OEP now: https://github.com/edx/open-edx-proposals/pull/156